.github/workflows/ci-e2e.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
       - uses: pnpm/action-setup@c3b53f6a16e57305370b4ae5a540c2077a1d50dd # tag=v2.2.4
         with:
-          version: 7
+          version: 7.27.0
```

Comment:
for these CI could you also include the comments `version: 7.27.0 # renovate: datasource=npm depName=pnpm`